setup.py

```diff
@@ -35,25 +24,27 @@
             continue
 
 
+install_requires = [
+    'httpx>=0.16',
+    'tqdm>=4.56',
+    'pywin32 >= 1.0;platform_system=="Windows"'
+]
+
 test_requires = [
     'pytest',
     'pytest-asyncio',
-    'respx'
+    'pytest-cov',
+    'respx==0.16.3'
 ]
 
-dev_requires = [
+lint_requires = [
     'flake8',
-    'setuptools',
-    'pex',
-    'pytest-cov',
-    'sphinx',
-    'wheel',
 ]
 
 setup(name='planet',
       version=version,
       description=u"Planet API Client",
-      long_description=long_description,
+      long_description=Path("README.md").read_text("utf-8"),
```

Comment:
yep, [README.md](https://github.com/planetlabs/planet-client-python/blob/v2-nox-243/README.md) exists in v2 and in this v2 subbranch. That does bring up the point that it would be nice to have the packaging process tested as a part of CI. Created #263